faust_s3_backed_serializer/s3_serializer.py

```diff
@@ -79,20 +79,26 @@ def __init__(self, output_topic: str, base_path: str, region_name: str, s3_crede
         ------
         ValueError
             If 'base_path' param is empty or not given
+            If 'output_topic' params is empty or not given
+
         """
 
         super().__init__(output_topic=output_topic, base_path=base_path, max_size=max_size,
                          s3_credentials=s3_credentials, region_name=region_name,
                          **kwargs)
 
+        self._base_path = base_path
+        self._max_size = max_size
+        self._is_key = is_key
+        self._output_topic = output_topic
+        self._verify_params()
```

Comment:
Only check this if dumps is called and message is too large

Reply:
what about now?

Reply:
Looks good.Please update the constructor doc

Reply:
the types right ?

Reply:
And the Exception part :)

Reply:
like this ?